Gemfile

```diff
@@ -7,6 +7,7 @@ gemspec
 gem("minitest")
 gem("minitest-hooks")
 gem("minitest-reporters")
+gem("debug", require: false)
```

Comment:
Shouldn't it be in the `development` group?

Reply:
It should at least be included in `test` as well?
I think it belongs to where `pry-byebug` sits, which is at the top-level atm.